azurerm/internal/services/storage/storage_account_resource.go

```diff
@@ -267,6 +267,41 @@ func resourceStorageAccount() *schema.Resource {
 								},
 							},
 						},
+
+						"change_feed": {
+							Type:     schema.TypeList,
+							Optional: true,
+							MaxItems: 1,
+							Elem: &schema.Resource{
+								Schema: map[string]*schema.Schema{
+									"retention_in_days": {
+										Type:         schema.TypeInt,
+										Optional:     true,
+										ValidateFunc: validation.IntBetween(1, 146000),
+									},
+								},
+							},
+						},
+
+						"versioning_enabled": {
+							Type:     schema.TypeBool,
+							Optional: true,
+							Default:  false,
+						},
+
+						"default_service_version": {
+							Type:         schema.TypeString,
+							Optional:     true,
+							Default:      "2020-06-12",
+							ValidateFunc: validate.BlobPropertiesDefaultServiceVersion,
+						},
+
+						"last_access_time_tracking_policy_enabled": {
```

Comment:
i think we can shorten this to
```suggestion
						"last_access_time_enabled": {
```

tracking and policy are redundant i think?